package.json

```diff
@@ -14,9 +14,9 @@
   "scripts": {
     "start": "cd infrastructure/develop && ./compose-build.sh && ./run-servers.sh dev-build",
     "prod-start": "cd infrastructure/develop && ./compose-build.sh && ./run-servers.sh prod-build",
-    "build": "cd frontend && CI=false react-scripts build",
+    "build": "cd frontend && CI=false react-app-rewired build",
     "test": "jest",
-    "eject": "cd frontend && react-scripts eject",
+    "eject": "cd frontend && react-app-rewired eject",
```

Comment:
I should have paid attention to this in the previous PR, but why do we need to use `react-app-rewired` ?

Reply:
its a library to do relative imports outside of src in Create-React-App without ejecting CRA and losing all those perks

Reply:
gotcha, makes sense